examples/train_val_test_split_example.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9feb8e56",
+   "metadata": {},
+   "source": [
+    "# Using `train_val_test_split` for Rigorous Modeling\n",
+    "Thanks to [this](https://towardsdatascience.com/automatic-hyperparameter-tuning-with-sklearn-gridsearchcv-and-randomizedsearchcv-e94f53a518ee) article for providing some code used below in the automatic hyperparameter tuning.\n",
+    "Load example data from sklearn"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "91321870",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn import datasets\n",
+    "diabetes_X, diabetes_y = datasets.load_diabetes(return_X_y=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e884e357",
+   "metadata": {},
+   "source": [
+    "Split the data into training, testing, and validation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "be4de1fa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astartes import train_val_test_split\n",
+    "X_train, X_val, X_test, y_train, y_val, y_test = train_val_test_split(diabetes_X, diabetes_y)"
```

Comment:
I'd potentially suggest using the default keyword argument of `sampler="random"` just so users are aware that this example uses random sampling. Same for the train, val, and test size so that someone quickly reading can see another example of the syntax in case they skimmed the readme. Or maybe a quick comment describing these would be better than specifying the arguments. What are your thoughts?

Reply:
Good to have a mix of the two -- will update now